### ISSUE TITLE
How does "ProjectID" differ from permit number

### ISSUE THREAD

**Issue by mheadd** [open-data-standards/permitdata.org#28]
Describe how this would be different to than the permit number or master permit number. Assuming this would be a "planning project" under which multiple permits could be issued for such as a Planned Unit Development or similar.

**Comment by stevevance** [comment]
Correct, @mheadd. A relevant example is how the City of Chicago issues business licenses. 

Consider the Target department store. All Target stores in Chicago are (likely) corporate owned, and thus share a single account number. Each Target store then has a site number. Then each store has several licenses (pharmacy, maybe motor vehicle, liquor, and a regular ol' limited business license). Each of those licenses has its own number and that license is renewed the renewed license retains the license number but gets a unique license ID. 

To recap, account_number>site_number>license_number>license_id. 

So, in the case of building permits there could be a project identifier that is a grouping of permit identifiers. Chicago doesn't do this in [its open data set](https://data.cityofchicago.org/Buildings/Building-Permits/ydr8-5enu) (from the Socrata-powered portal) but it may have an internal identifier. [Chicago Cityscape](http://www.chicagocityscape.com) uses the address as the connection among permits.

**Comment by aditya180** [comment]
Seems like a great explanation.  Thanks for that @stevevance.  We will leave ProjectID in.  Thanks.
